src/games/supported/Zaxxon.cpp

```diff
@@ -132,3 +132,25 @@ void ZaxxonSettings::loadState(Deserializer & ser) {
   m_lives = ser.getInt();
 }
 
+// returns a list of mode that the game can be played in
+ModeVect ZaxxonSettings::getAvailableModes() {
+    ModeVect modes = {0, 8, 16, 24};
+    return modes;
+}
+
+// set the mode of the game
+// the given mode must be one returned by the previous function
+void ZaxxonSettings::setMode(game_mode_t m, System &system,
+                              std::unique_ptr<StellaEnvironmentWrapper> environment) {
+
+    // read the mode we are currently in
+    unsigned char mode = readRam(&system, 0x82);
+    // press select until the correct mode is reached
+    while (mode != m) {
+        environment->pressSelect(10);
```

Comment:
Worth commenting on '10'.

Reply:
I'm not sure what to say here. Each game requires an arbitrary number of presses to the select button. This one is `10`, James Bond is `20`, Freeway is `1`.

Reply:
Just a comment to indicate this isn't a typo. "Hold select button for 10 frames".

Reply:
Ah, ok. Sure. I'll merge it now.